scripts/update-submodule.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Update python-base submodule and collect release notes.
+# Usage:
+#   scripts/update-submodule.sh
+#
+# Create a commit "generated python-base update" and send a PR to the main repo.
+# NOTE: not all python-base changes are user-facing, and you may want to remove
+# some of the non-user-facing release notes from CHANGELOG.md.
```

Comment:
is there a release-note label in PR similar to k/k? then it's the matter of automatically collecting all those notes w/o any non user-facing notes

Reply:
updated to use a release-note label. I'm going to add PR template to python-base and see if we can enable `release-note-required` in these repos.